include/mgos_ads1x1x.h

```diff
@@ -50,19 +51,41 @@ enum mgos_ads1x1x_dr {
   MGOS_ADS1X1X_SPS_MIN = 0, // 8SPS for ADS111X, 128SPS for ADS101X
   MGOS_ADS1X1X_SPS_8,       // 8SPS, ADS111X only
   MGOS_ADS1X1X_SPS_16,      // 16SPS, ADS111X only
+  MGOS_ADS1X1X_SPS_20,      // 20SPS, ADS1119 only (default)
   MGOS_ADS1X1X_SPS_32,      // 32SPS, ADS111X only
   MGOS_ADS1X1X_SPS_64,      // 64SPS, ADS111X only
+  MGOS_ADS1X1X_SPS_90,      // 90SPS, ADS1119 only
   MGOS_ADS1X1X_SPS_128,     // 128SPS, both ADS111X and ADS101X
   MGOS_ADS1X1X_SPS_250,     // 250SPS, both ADS111X and ADS101X
+  MGOS_ADS1X1X_SPS_330,     // 330SPS, ADS1119 only
   MGOS_ADS1X1X_SPS_475,     // 475SPS, ADS111X only
   MGOS_ADS1X1X_SPS_490,     // 490SPS, ADS101X only
   MGOS_ADS1X1X_SPS_860,     // 860SPS, ADS111X only
   MGOS_ADS1X1X_SPS_920,     // 920SPS, ADS101X only
+  MGOS_ADS1X1X_SPS_1000,    // 1000SPS, ADS1119 only
   MGOS_ADS1X1X_SPS_1600,    // 1600SPS, ADS101X only
   MGOS_ADS1X1X_SPS_2400,    // 2400SPS, ADS101X only
   MGOS_ADS1X1X_SPS_3300,    // 3300SPS, ADS101X only
-  MGOS_ADS1X1X_SPS_DEFAULT, // 128SPS for ADS111X, 1600SPS for ADS101X
-  MGOS_ADS1X1X_SPS_MAX,     // 860SPS for ADS111X, 3300SPS for ADS101X
+  MGOS_ADS1X1X_SPS_DEFAULT, // 128SPS for ADS111X, 1600SPS for ADS101X, 20SPS for ADS1119
+  MGOS_ADS1X1X_SPS_MAX,     // 860SPS for ADS111X, 3300SPS for ADS101X, 1000SPS for ADS1119
+};
+
+enum mgos_ads1119_gain
+{
+    MGOS_ADS1119_GAIN_1 = 0,
+    MGOS_ADS1119_GAIN_4
+};
+
+enum mgos_ads1119_conversion_mode
+{
+    MGOS_ADS1119_CM_SS = 0,
+    MGOS_ADS1119_CM_CONT
+};
```

Comment:
singleshot vs continuous mode exists also in some other versions in the ads1x1x family, so this can be a shared enum, probably hidden behind an `ads1x1x_{get,set}_mode()` API;